extensions/2.0/Khronos/KHR_spz_gaussian_splats_compression/README.md

```diff
@@ -0,0 +1,437 @@
+
+## Adding Gaussian Splats to Primitives
+
+If a primitive contains an `extension` property which defines `KHR_spz_gaussian_splats_compression` then support for SPZ compression is required. At this time, there is no requirement for a backup uncompressed buffer.
+
+The extension must be listed in `extensionsUsed`
+
+```json
+  "extensionsUsed" : [
+    "KHR_spz_gaussian_splats_compression"
+  ]
+```
+
+It must also be listed in `extensionsRequired`
+
+```json
+  "extensionsRequired" : [
+    "KHR_spz_gaussian_splats_compression"
+  ]
+```
+
+Note that while `KHR_spz_gaussian_splats_compression` is required.
+
+### Geometry Type
+
+The `mode` of the `primitive` must be `POINTS`.
+
+### Schema Example
+
+Example SPZ extension shown below. This extension only affects any `primitive` nodes containting Gaussian splat data.
+
+```json
+  "meshes": [{
+      "primitives": [{
+          "attributes": {
+            "_ROTATION": 3,
+            "COLOR_0": 1,
+            "_SCALE": 2,
+            "POSITION": 0,
+            "_SH_DEGREE_1_COEF_0": 4,
+            "_SH_DEGREE_1_COEF_1": 5,
+            "_SH_DEGREE_1_COEF_2": 6,
+          },
+          "material": 0,
+          "mode": 0,
+          "extensions": {
+            "KHR_spz_gaussian_splats_compression": {
+              "bufferView": 0,
+              "numPoints": 590392,
+              "shDegree": 0,
+              "fractionalBits": 12,
+              "flags": 0,
+              "attributes": {
+                "_ROTATION": 3,
+                "COLOR_0": 1,
+                "_SCALE": 2,
+                "POSITION": 0,
+                "_SH_DEGREE_1_COEF_0": 4,
+                "_SH_DEGREE_1_COEF_1": 5,
+                "_SH_DEGREE_1_COEF_2": 6,
+              }
+            }
+          }
+        }]
+    }],
+  "buffers": [{
+      "uri": "0.spz",
+      "byteLength": 9753142
+    }],
+  "bufferViews": [{
+      "buffer": 0,
+      "byteLength": 9753142
+    }],
+```
+
+### Extension Properties
+
+#### bufferView
+
+This property points to the bufferView containing the Gaussian splat data compressed with SPZ.
+
+#### attributes
+
+This contains the attributes that will map into the compressed SPZ data indicated by `bufferView`. At minimum it will contain `POSITION`, `COLOR_0`, `_ROTATION`, and `_SCALE`. `_SH_DEGREE_ℓ_COEF_n` attributes hold the spherical harmonics data and are not required. If higher degrees are used then lower degrees are required implicitly.
+
+| Splat Data | glTF Attribute | Accessor Type | Component Type | Required |
+| --- | --- | --- | --- | --- |
+| Position | POSITION | VEC3 | float | yes |
+| Color (Spherical Harmonic degree 0 (Diffuse)) | COLOR_0 | VEC4 | unsigned byte or float normalized | yes |
+| Rotation | _ROTATION | VEC4 | float | yes |

```

Comment:
I'm not sure if it's mentioned anywhere that `_ROTATION` is a quaternion

Reply:
It isn't, but it should be. I've fixed it.